lib/src/widget/nav_bar/widget/nav_bar_item/edit.dart

```diff
@@ -22,27 +22,27 @@ class NavBarEdit extends StatefulWidget {
 
 class _NavBarEditState extends State<NavBarEdit> {
   late final TextEditingController controller;
+  bool _didAutoFocus = false;
 
   Linden get linden => UnterDenLinden.getLinden(context);
 
   @override
   void initState() {
-    controller = TextEditingController(text: widget.data.initialText);
-    if (widget.data.autofocus) {
-      WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {
-        widget._focusNode.requestFocus();
-      });
-    }
+    controller = TextEditingController(
+      text: widget.data.initialText,
+    );
 
     widget._focusNode.addListener(() {
       widget._updateNavBar(context);
     });
+
     super.initState();
   }
 
   @override
   void dispose() {
     controller.dispose();
+    widget._focusNode.dispose();
```

Comment:
❓ we didn't create `node` in this class, so it's not `our` responsibility to close it here. wdyt